x-pack/plugins/actions/server/builtin_action_types/jira/service.ts

```diff
@@ -103,6 +104,13 @@ export const createExternalService = (
     return fields;
   };
 
+  const trimAndRemoveNewlines = (str: string) =>
+    str
+      .split(/[\n\r]/gm)
+      .filter((item) => !isEmpty(item.trim()))
```

Comment:
nit: if we swap the two operations we can get away with the second `trim` call.

```
.map((item) => item.trim())
.filter((item) => !isEmpty(item))
```

Reply:
good point, @jamster10 I'd add Christos' suggestion.

Reply:
sorted 👌